lib/tls.go

```diff
@@ -92,27 +93,37 @@ var tlsVersions = map[uint16]description{
 	tls.VersionTLS12: {"TLS 1.2", "tls_1_2", 2},
 }
 
+// Fill in a human readable name, extracted from the slug
+func explain_cipher(d description) description {
```

Comment:
Nit: no underscores in Go. Use camelCase. Same for kex_cipher, encrypt_auth below.